uint32 angle
float pose[]
---
bool obstacle_clearance 
---
bool obstacle_presence
